core/renderers/common/info.js

```diff
@@ -10,37 +10,37 @@
-goog.require('Blockly.blockRendering.JaggedEdge');
-goog.require('Blockly.blockRendering.Measurable');
-goog.require('Blockly.blockRendering.NextConnection');
-goog.require('Blockly.blockRendering.OutputConnection');
-goog.require('Blockly.blockRendering.PreviousConnection');
-goog.require('Blockly.blockRendering.RoundCorner');
-goog.require('Blockly.blockRendering.Row');
-goog.require('Blockly.blockRendering.SpacerRow');
-goog.require('Blockly.blockRendering.SquareCorner');
-goog.require('Blockly.blockRendering.StatementInput');
-goog.require('Blockly.blockRendering.TopRow');
-goog.require('Blockly.blockRendering.Types');
+goog.module('Blockly.blockRendering.RenderInfo');
+goog.module.declareLegacyNamespace();
+
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const BottomRow = goog.require('Blockly.blockRendering.BottomRow');
+const ConstantProvider = goog.requireType('Blockly.blockRendering.ConstantProvider');
+const ExternalValueInput = goog.require('Blockly.blockRendering.ExternalValueInput');
+const Field = goog.require('Blockly.blockRendering.Field');
+const Hat = goog.require('Blockly.blockRendering.Hat');
+const Icon = goog.require('Blockly.blockRendering.Icon');
+const InlineInput = goog.require('Blockly.blockRendering.InlineInput');
+const Input = goog.requireType('Blockly.Input');
+const InputRow = goog.require('Blockly.blockRendering.InputRow');
+const inputTypes = goog.require('Blockly.inputTypes');
+const InRowSpacer = goog.require('Blockly.blockRendering.InRowSpacer');
+const JaggedEdge = goog.require('Blockly.blockRendering.JaggedEdge');
+const Measurable = goog.require('Blockly.blockRendering.Measurable');
+const NextConnection = goog.require('Blockly.blockRendering.NextConnection');
+const OutputConnection = goog.require('Blockly.blockRendering.OutputConnection');
+const PreviousConnection = goog.require('Blockly.blockRendering.PreviousConnection');
+const RenderedConnection = goog.requireType('Blockly.RenderedConnection');
+const Renderer = goog.requireType('Blockly.blockRendering.Renderer');
+const RoundCorner = goog.require('Blockly.blockRendering.RoundCorner');
+const Row = goog.require('Blockly.blockRendering.Row');
+const SpacerRow = goog.require('Blockly.blockRendering.SpacerRow');
+const SquareCorner = goog.require('Blockly.blockRendering.SquareCorner');
+const StatementInput = goog.require('Blockly.blockRendering.StatementInput');
+const TopRow = goog.require('Blockly.blockRendering.TopRow');
+const Types = goog.require('Blockly.blockRendering.Types');
 /** @suppress {extraRequire} */
-goog.require('Blockly.constants');
-goog.require('Blockly.inputTypes');
-
-goog.requireType('Blockly.blockRendering.ConstantProvider');
-goog.requireType('Blockly.blockRendering.Icon');
-goog.requireType('Blockly.blockRendering.Renderer');
-goog.requireType('Blockly.BlockSvg');
-goog.requireType('Blockly.Input');
-goog.requireType('Blockly.RenderedConnection');
+const blocklyConstants = goog.require('Blockly.constants');
```

Comment:
Non-blocking: There's been a lot of discussion on this.

I think it would be ideal if this was:
```
const {ALIGN} = goog.require('Blockly.constants'); 
```
For the usage in this file.

However, as discussion offline, this file has side effects so I'm ok with it being left as is and changed later when we deal with the `core/constants.js` file.

## Alternative approach
I'm not sure if this is valid, but maybe we could do something like:
```
const {ALIGN} = goog.require('Blockly.constants');
// ...
/** @suppress {extraRequire} */
goog.require('Blockly.constants');
```

That way we ensure we ensure we don't forget to do the destructuring and still ensure we have the extra require.

Reply:
Moved to the first version.